Android App/app/src/main/java/org/cfp/citizenconnect/MyUtils.java

```diff
@@ -91,7 +91,7 @@ public static Bitmap getBitmap(int drawable, Context mContext) {
 
     @WorkerThread
     public static void getAFireBaseData(DatabaseReference databaseReference, CustomCallBack.Listener<DataSnapshot> mListener, CustomCallBack.ErrorListener<DatabaseError> mErrorListener) {
-        databaseReference.addValueEventListener(new ValueEventListener() {
+        databaseReference.addListenerForSingleValueEvent(new ValueEventListener() {
```

Comment:
I already created a separate function for `addListenerForSingleValueEvent()` in the PR which I created. But if you are sure about this then whoever merges later should fix this as part of conflict resolution.

Reply:
will keep this in mind